Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:16.04
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends apt-utils git \
+                            libatlas-base-dev python-protobuf python-numpy \
+                            python-scipy python-h5py unzip make libblas-dev \
+                            liblapack-dev libatlas-base-dev gfortran \
+                            python-pip python-dev libprotobuf-dev \
+                            libleveldb-dev libsnappy-dev libopencv-dev \
+                            libhdf5-serial-dev libgflags-dev nodejs npm \
+                            libgoogle-glog-dev liblmdb-dev protobuf-compiler \
+                            cmake libboost-all-dev wget python-setuptools g++ \
+                            && rm -rf /var/lib/apt/lists/*
+
+ADD requirements/dev.txt requirements/common.txt /tmp/
+
+RUN pip install setuptools wheel \
+    && pip install numpy scipy scikit-image \
+    && pip install -r /tmp/dev.txt \
+    && pip install --upgrade keras==2.0.7 https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.12.1-cp27-none-linux_x86_64.whl
+
+RUN mkdir /caffe && cd /caffe \
```

Comment:
It might be a good idea to look for Docker images that come with prebuilt caffe so that we don't have to worry about installing in the container? Do you see what I mean?

Reply:
Nice idea ! Looks like there are some already prebuileded images with caffe. Let me try to use some.